docker-images/test-client/Dockerfile

```diff
@@ -1,17 +1,9 @@
 FROM strimzi/kafka:latest
 
-ARG strimzi_version=1.0-SNAPSHOT
-ENV STRIMZI_VERSION ${strimzi_version}
 ENV JAVA_OPTS "-DLOG_LEVEL=info"
 
 # copy scripts for starting Kafka
 COPY scripts $KAFKA_HOME
 COPY scripts/dynamic_resources.sh /bin/dynamic_resources.sh
 
-ADD tmp/test-client-${STRIMZI_VERSION}.jar /test-client.jar
-
-USER 1001
-
-EXPOSE 4242/tcp
-
-CMD ["/opt/kafka/launch_java.sh", "/test-client.jar"]
\ No newline at end of file
+USER 1001
```

Comment:
It looks like you're pretty much deleting the test-client module, but for some reason you're keeping the `Dockerfile` and `Makefile`. Why?

Reply:
We removed `test-client` package, but `test-client` image is still needed for the tests with our "tooling" scripts for producers and consumers